crypto/src/main/java/org/web3j/crypto/Async.java

```diff
@@ -0,0 +1,16 @@
+package org.web3j.crypto;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class Async {
```

Comment:
I'd rather not have a sync helper method in all of this. My preference is to have all of the core code paths taking place in a synchronous manner by default, without additional futures in the mix.

Besides, there is already an Async helper class in the project - see https://github.com/web3j/web3j/blob/master/core/src/main/java/org/web3j/utils/Async.java. Appreciate it doesn't cater for this specific functionality, but these utils should only be defined in one place.

Reply:
On the existing class, I did notice it, but I realised it would add an extra dependency to `crypto` and I wasn't sure if it was desired or not. I can certainly change that.